tests/plugins/mocha.test.support/hooks/hook.js

```diff
@@ -32,24 +33,66 @@ exports.init = (logger, config, cli) => {
 
 	cli.on('build.post.compile', async (builder, done) => {
 		if (builder.platformName === 'android') {
-			await wakeDevices(logger, builder).catch(e => logger.warn(`could not wake ${builder.deviceId}: ${e}`));
+			const emulatorManager = new EmulatorManager(config);
+
+			await wakeDevices(logger, builder, emulatorManager)
+				.catch(e => logger.warn(`Could not wake ${builder.deviceId}: ${e}`));
+			await enableAutoRotation(logger, builder, emulatorManager)
+				.catch(e => logger.warn(`Could not enable auto-rotation ${builder.deviceId}: ${e}`));
 		}
 		done();
 	});
 };
 
-async function adb(argumentArray) {
-	return new Promise(resolve => appc.subprocess.run(ADB_PATH, argumentArray, { shell: false, windowsHide: true }, (error, stdout, _stderr) => {
-		resolve(stdout);
-	}));
+async function adb(args) {
+	return new Promise((resolve, reject) => {
+		let child;
+		let stdout = '';
+		let stderr = '';
+
+		if (process.platform === 'win32') {
+			child = spawn(
+				process.env.comspec || 'cmd.exe',
+				[ '/S', '/C', '"', ADB_PATH, ...args, '"' ],
+				{
+					windowsVerbatimArguments: true
+				}
+			);
+		} else {
+			child = spawn(
+				ADB_PATH,
+				args,
+				{
+					shell: false,
+					windowsHide: true
+				}
+			);
+		}
+		if (child) {
+			child.stdout.on('data', data => {
+				stdout += data.toString();
+			});
+			child.stderr.on('data', data => {
+				stderr += data.toString();
+			});
+			child.on('close', code => {
+				if (code !== 0) {
+					return reject(`${stdout}\n${stderr}`);
+				}
+				resolve(stdout);
+			});
+			return;
+		}
+		reject();
+	});
 }
 
-async function wakeDevices(logger, builder) {
+async function wakeDevices(logger, builder, emulatorManager) {
 
 	async function wake(device) {
 		logger.info(`Waking up ${device}`);
 
-		const deviceId = device !== 'emulator' ? [ '-s', device ] : [];
+		const deviceId = device !== 'emulator' ? [ '-s', await getDeviceId(device, emulatorManager) ] : [];
```

Comment:
This change appears to have broken the part of the build where it wakes the emulator. It's just sitting there at this point until the build times out.